src/containers/ProjectDashboardContainer.jsx

```diff
@@ -44,18 +44,18 @@ class ProjectDashboardContainer extends Component {
 
   handleSelect({ option }) {
     this.setState({
-      translationLanguage: option,
+      translationLanguage: option.value,
     });
   }
 
   getSelectOptions() {
     const searchStr = this.state.searchText;
     let langs = languages.slice();
     if (searchStr) {
-      const matchesSearch = (langObj) => langObj.name.toLowerCase().substr(0, searchStr.length) === searchStr;
+      const matchesSearch = (langObj) => langObj.label.toLowerCase().substr(0, searchStr.length) === searchStr;
       langs = languages.filter(matchesSearch);
     }
-    return langs.map(item => item.name);
+    return langs.map(item => item);
```

Comment:
This maps `langs` back to a copy of itself. I think you just mean `return langs`.